packages/jest-transform/src/shouldInstrument.ts

```diff
@@ -58,7 +58,7 @@ export default function shouldInstrument(
   if (
     // still cover if `only` is specified
     !options.collectCoverageOnlyFrom &&
-    options.collectCoverageFrom &&
+    options.collectCoverageFrom.length &&
```

Comment:
sorta breaking? Only for people calling this manually though, and they'd get a type error. Fine for Jest 25.

Also a bug fix if you forgot to send it in

Reply:
well, Jest didn't always pass it, so that's nice :P Looking forward to making `normalize` in `jest-config` actually be type safe...